src/Server/middleware/userRequiredMiddleware.ts

```diff
@@ -1,6 +1,7 @@
 import { match } from "path-to-regexp"
 
 const USER_REQUIRED_ROUTES = [
+  "/notifications",
```

Comment:
if a user who is not logged in tries to open the page, we will redirect to the log in page